src/singletons/Theme.cpp

```diff
@@ -48,7 +48,7 @@ void Theme::actuallyUpdate(double hue, double multiplier)
         this->splits.resizeHandleBackground = QColor(0, 148, 255, 0x20);
     }
 
-    this->messages.backgrounds.highlighted = QColor(140, 84, 89, 127);
+    this->messages.backgrounds.highlighted = QColor(127, 63, 73, 127);
```

Comment:
Why was this color changed?

Reply:
When you mix this color (alpha = 127) with the default background it gets you exactly the old highlight color

Reply:
Also see https://github.com/Chatterino/chatterino2/issues/1565#issuecomment-590441625.

Reply:
So it works for light+dark backgrounds?

Reply:
Light theme:
![image](https://user-images.githubusercontent.com/1629196/75632585-cc42ba00-5bfd-11ea-88a8-e4e5dccfbe5d.png)

White theme:
![image](https://user-images.githubusercontent.com/1629196/75632589-d8c71280-5bfd-11ea-876c-dd499a23a8e3.png)

Dark theme:
![image](https://user-images.githubusercontent.com/1629196/75632595-e2e91100-5bfd-11ea-8776-f7550ca691ae.png)

Black theme:
![image](https://user-images.githubusercontent.com/1629196/75632597-ec727900-5bfd-11ea-80b6-ce32069814e0.png)

Depends on your taste whether you think it's good.

Reply:
In any case, it would probably be "better" as it is very close, if not identical, to the original highlight color in the dark theme (which is what most people use, I suppose). Guess it would minimize irritation when users are migrated to the new highlight system.

Reply:
That's why I suggested the color in the first place. Every existing user is going to go through the migration phrase, and you risk a bad user experience if their old experience does not translate to the new system transparently. (I can already imagine the waves of whining users - "This update sucks.")